requests/PermissionRequest.go

```diff
@@ -0,0 +1,67 @@
+package requests
+
+// PermissionRequest structure to change permission
+type PermissionRequest struct {
+	Permissions [][]int
+}
+
+// PermissionRequestBuilder structure to create permission permissionRequest
+type PermissionRequestBuilder struct {
+	permissionRequest PermissionRequest
+}
+
+// PermissionGroup type
+type PermissionGroup int
+
+const (
+	// User of PermissionGroup
+	User PermissionGroup = iota
+	// Group of PermissionGroup
+	Group
+	// Other of PermissionGroup
+	Other
+)
+
+// PermissionType type
+type PermissionType int
+
+const (
+	// Use of PermissionType
+	Use PermissionType = iota
+	// Manage of PermissionGroup
+	Manage
+	// Admin of PermissionGroup
+	Admin
+)
+
+// Allow to change permission
```

Comment:
These comments (both for constants and on methods) are misleading. `Allow` sets certain permission type for certain permission group. For example I'm allowing a **group** of a resource to **use** said resource. `Deny` does the opposite. For example I'm denying a **user** (owner) of a resource to **manage** said resource. `NoChange` just keeps the setting as it is. There is no default.